python/gtsam_py/python/utils/visual_data_generator.py

```diff
@@ -1,17 +1,17 @@
 from __future__ import print_function
 
 import numpy as np
-
+from math import pi, cos, sin
```

Comment:
Add 2.7 to see what the work is... I vote for test being dual-support, but example be 3.X -> then remove all __future__ in the examples.